Sequential commit

zajck opened the pull request:
This PR describes a sequential commit proposal.

thecryptofruit reviewed (approved):
For completeness, I suggest to include a note on the case when the secondary sales have smaller-or-equal price than the original.

zajck commented:
> For completeness, I suggest to include a note on the case when the secondary sales have smaller-or-equal price than the original.

Added. 
I also added a paragraph multiple sequential commit for one voucher and I fixed some typos.
